wrangler-proto/src/main/java/io/cdap/wrangler/proto/workspace/v2/Workspace.java

```diff
@@ -79,6 +83,15 @@ public JsonObject getInsights() {
     return insights;
   }
 
+  public HashMap<String, UserDefinedAction> getColumnMappings() {
```

Comment:
Can we keep the same name here too or make it more intuitive that column mappings refer to the nullability map

Reply:
Thanks for pointing that out. I missed that.